themes/aqua.typ

```diff
@@ -0,0 +1,308 @@
+  self.page-args = self.page-args + (
+    margin: (x:0em, y:10%),
+  )
+  self.padding = self.padding + (x: 5%, top:1em)
+  (self.methods.slide)(self:self, title: if self.aqua-lang=="zh" {[总结]} else if self.aqua-lang=="en" {[Summary]}, setting: body =>{
+    if self.aqua-lang == "zh" {
+      v(15%)
+      stack(
+        spacing: 20%,
+        align(
+          center,
+          text(size:70pt,fill:self.colors.primary-light,weight: "bold")[THANKS FOR ALL]
+        ),
+        align(
+          center,
+          text(size:60pt,fill:self.colors.primary-light,weight: "bold")[敬请指正！]
+        )
+      )
+    }else if self.aqua-lang == "en" {
+      v(40%)
+      align(
+          center,
+          text(size:70pt,fill:self.colors.primary-light,weight: "bold")[THANKS FOR ALL]
+      )
+    }
+  })
+}
+
+#let slides(self: none, title-slide: true, slide-level: 1, ..args) = {
+  if title-slide {
+    (self.methods.title-slide)(self: self)
+  }
+  (self.methods.touying-slides)(self: self, slide-level: slide-level, ..args)
+}
+
+#let register(
+  aspect-ratio: "16-9",
+  aqua-lang: "en",
+  self,
+) = {
+  self = (self.methods.colors)(
+    self: self,
+    primary: rgb(0, 63, 136),
+    primary-light: rgb(33,89,165),
+    secondary: rgb(242,244,248),
+  )
+
+
+  self.slide-title = []
+  self.section_num = counter("section")
+  self.aqua-lang = aqua-lang
+  self.background = (self)=> {
+  place(left + top, dx: -15pt, dy: -26pt,
+    circle(radius: 40pt, fill: self.colors.primary,))
+  place(left + top, dx: 65pt, dy: 12pt,
+    circle(radius: 21pt, fill: self.colors.primary,))
+  place(left + top, dx: 3%, dy: 15%,
+    circle(radius: 13pt, fill: self.colors.primary,))
+  place(left + top, dx: 2.5%, dy: 27%,
+    circle(radius: 8pt, fill: self.colors.primary,))
+  place(right + bottom, dx: 15pt, dy: 26pt,
+    circle(radius: 40pt, fill: self.colors.primary,))
+  place(right + bottom, dx: -65pt, dy: -12pt,
+    circle(radius: 21pt, fill: self.colors.primary,))
+  place(right + bottom, dx: -3%, dy: -15%,
+    circle(radius: 13pt, fill: self.colors.primary,))
+  place(right + bottom, dx: -2.5%, dy: -27%,
+    circle(radius: 8pt, fill: self.colors.primary,))
+  polygon(fill: self.colors.secondary,
+    (35%, -17%), (70%, 10%), (35%, 30%), (0%, 10%),)
+  place(center + horizon, dy: 7%,
+    ellipse(fill: white, width: 45%, height: 120pt))
+  place(center + horizon, dy: 5%,
+    ellipse(fill: self.colors.secondary, width: 40%, height: 80pt))
+  place(center + horizon, dy: 12%,
+    rect(fill: self.colors.secondary, width: 40%, height: 60pt))
+  place(center + horizon, dy: 20%,
+    ellipse(fill: white, width: 40%, height: 70pt))
+  place(center + horizon, dx: 28%, dy: -6%,
+    circle(radius: 13pt, fill: white,) )
+}
+
+  let footer(self) = {
+  set text(size:0.8em)
+  place(
+    right,
+    dx: -5%, 
+    [#counter(page).display("1")]
+  )
+}
+
+  let header(self)={
+  place(
+    center+top,
+    dy: 45%,
+    image("./images/header.png", width: 100%)
```

Comment:
Why is there a `header.png` here?

Reply:
Oh, it must be because I forgot to delete that part, I'll fix it right away.